agent/apps/ai_guardrails/app_code/serializers.py

```
@@ -0,0 +1,93 @@
+    سریالایزر سیاست گاردریل
+    """
+
+    class Meta:
+        model = GuardrailPolicy
+        fields = [
+            'id', 'name', 'description', 'is_active', 'enforcement_mode',
+            'applies_to', 'priority', 'conditions', 'created_at', 'updated_at', 'created_by'
+        ]
+        read_only_fields = ['id', 'created_at', 'updated_at', 'created_by']
+
+    def validate_conditions(self, value):
```

Ruff (0.12.2)

Missing return type annotation for public function `validate_conditions`

(ANN201)